.github/actions/lambda_deploy/action.yaml

```diff
@@ -24,11 +24,24 @@ runs:
       with:
         poetry-dir: py_gtfs_rt_ingestion
 
+    - name: Download psycopg2 with libpq installed
+      uses: actions/checkout@v2
+      with:
+        # this repo has a bunch of stars and bundles psycopg with libpq
+        repository: jkehler/awslambda-psycopg2
+        path: psycopg_with_libpq
+
```

Comment:
i tried for a while to build the thing ourselves so that we could select what versions of postgres and psycopg we're building with, but was unsuccesful as the containers on github actions are missing some things we would need. we could try to build our own artifacts on ubuntu later and use them instead. this seems to work ok for now though.